ui/app/serializers/identity/entity-alias.js

```diff
@@ -4,4 +4,29 @@
  */
 
 import IdentitySerializer from './_base';
-export default IdentitySerializer.extend();
+export default IdentitySerializer.extend({
+  extractLazyPaginatedData(payload) {
```

Comment:
We have to serialize the data here so that the information is sortable by name in lazyPaginatedQuery